Cannot find module 'lib/imageRouter/minio'

A bug appeared after upgrading to CodiMD 2.0.0

I am now using this docker image ```hackmdio/hackmd:2.0.0```

It seems my codimd can't connect to my minio instance anymore.
It used to work when i was using ```hackmdio/hackmd:1.4.1```

It seems some dependencies is missing. Easy fix ?

```
2020-03-11T19:59:06.876Z error: 	uncaughtException: Cannot find module 'lib/imageRouter/minio'
Error: Cannot find module 'lib/imageRouter/minio'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/hackmd/app/lib/imageRouter/minio.js:9:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at /home/hackmd/app/lib/imageRouter/index.js:30:30
    at IncomingForm.<anonymous> (/home/hackmd/app/node_modules/formidable/lib/incoming_form.js:107:9)
    at emitNone (events.js:106:13)
    at IncomingForm.emit (events.js:208:7)
    at IncomingForm._maybeEnd (/home/hackmd/app/node_modules/formidable/lib/incoming_form.js:557:8)
    at /home/hackmd/app/node_modules/formidable/lib/incoming_form.js:238:12
    at WriteStream.<anonymous> (/home/hackmd/app/node_modules/formidable/lib/file.js:79:5)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at WriteStream.emit (events.js:208:7)
2020-03-11T19:59:06.876Z error: 	An uncaught exception has occured.
2020-03-11T19:59:06.876Z error: 	Cannot find module 'lib/imageRouter/minio'
2020-03-11T19:59:06.876Z error: 	Process will exit now.
```

--- a/lib/imageRouter/minio.js
+++ b/lib/imageRouter/minio.js
@@ -6,7 +6,7 @@ const config = require('../config')
 const { getImageMimeType } = require('../utils')
 const logger = require('../logger')
 
-const Minio = require('lib/imageRouter/minio')
+const Minio = require('minio')
 const minioClient = new Minio.Client({
   endPoint: config.minio.endPoint,
   port: config.minio.port,

